src/pages/LoginPage.js

```diff
@@ -0,0 +1,393 @@
+    window.location.replace(githubAuthUrl.href)
+  }
+
+  return <>
+    <Typography
+      width='80%'
+      whiteSpace='wrap'
+      color='grey'
+      textAlign='center'
+    >
+      Press Login to be redirected to the selected OAuth Connect Provider.
+    </Typography>
+    <OAuthConfigSelector
+      selectedAuthConfig={selectedAuthConfig}
+      setSelectedAuthConfig={setSelectedAuthConfig}
+      authConfigs={authConfigs}
+    />
+    <LoginButton login={() => login(selectedAuthConfig)}/>
+  </>
+}
+OAuthTab.displayName = 'OAuthTab'
+
+
+const OAuthConfigSelector = ({
+  selectedAuthConfig,
+  setSelectedAuthConfig,
+  authConfigs,
+}) => {
+  if (authConfigs?.length === 0) {
+    return <Alert severity='warning' sx={{
+      width: '80%',
+      marginTop: '1rem',
+    }}>
+      No configured OAuth provider found.
+    </Alert>
+  }
+
+  if (!authConfigs || !selectedAuthConfig) {
+    return <Skeleton style={{
+      width: '80%',
+      marginTop: '1rem',
+    }}/>
+  }
+
+  return <FormControl size='small' sx={{
+    width: '80%',
+    marginTop: '1rem',
+  }}>
+    <InputLabel>OAuth Provider Configuration</InputLabel>
+    <Select
+      label='OAuth Provider Configuration'
+      value={selectedAuthConfig.name}
+      onChange={(e) => {
+        setSelectedAuthConfig(authConfigs.find((authConfig) => authConfig.name === e.target.value))
+      }}
+    >
+      {
+        authConfigs.map((authConfig) => <MenuItem
+          key={authConfig.name}
+          value={authConfig.name}
+        >
+          {
+            authConfig.name
+          }
+        </MenuItem>)
+      }
+    </Select>
+  </FormControl>
+}
+OAuthConfigSelector.displayName = 'OAuthConfigSelector'
+OAuthConfigSelector.propTypes = {
+  selectedAuthConfig: PropTypes.object,
+  setSelectedAuthConfig: PropTypes.func.isRequired,
+  authConfigs: PropTypes.arrayOf(PropTypes.object),
+}
+
+
+const TokenTab = ({
+  token,
+  setToken,
+  showToken,
+  setShowToken,
+}) => {
+  return <>
+    <Typography
+      width='80%'
+      whiteSpace='pre-wrap'
+      color='grey'
+      textAlign='center'
+    >
+      Enter a valid bearer token and press Login.
+    </Typography>
+    <TextField
+      onChange={(e) => setToken(e.target.value)}
+      value={token}
+      size='small'
+      sx={{
+        margin: '1rem 2rem 0 2rem',
+        width: '80%',
+      }}
+      label='Token'
+      type={showToken ? 'text' : 'password'}
+      InputProps={{
+        endAdornment: <InputAdornment position='end'>
+          <IconButton
+            onClick={() => setShowToken(!showToken)}
+            edge='end'
+            sx={{
+              color: 'grey',
+            }}
+            disableRipple
+          >
+            {
+              showToken ? <VisibilityOff/> : <Visibility/>
+            }
+          </IconButton>
+        </InputAdornment>
+      }}
+    />
+    <LoginButton login={() => { /* TODO implement login via token */ }}/>
```

Comment:
nit: it might be helpful to additionally write a warning to console.log to make it slightly more obvious. or even throw an error (let it crash the application - this is more feedback than a login-button that just does nothing). wdyt?

Reply:
This code path is never reached since the parent tab (`TokenTab`) is currently disabled (see other TODO).